assets/audio/sequences/seq_0.prg.seq

```diff
@@ -1,4 +1,5 @@
 #include "aseq.h"
+#include "versions.h"
```

Comment:
Could this be moved below the soundfont includes? It doesn't matter for anything but I think it's nicer

Reply:
sure, I thought it fit more alongside "aseq.h" personally (it's more "system-level" than the soundfont includes and it has the same capitalization as aseq.h)